exercises/IaC/ansible/variables/.templates/README.md.erb

```diff
@@ -0,0 +1,378 @@
+# Variables
+<%
+cic_connect_command  = 'cic connect'
+cic_up_command = 'cic up'
+cic_down_command = 'cic down'
+%>
+
+## Introduction
+Variables are crucial to allowing the Ansible we write to be reusable. The cost of maintaining code that is effectively duplicated barring values that make it specific to a given environment is a risky business. Snow blindness can easily set in and seemingly weird errors can occur if you you forget to rollout a change across all of the duplicate code. If only it was possible to inject the values that need to vary from one execution to the next... That's where variables come in.
+
+In this exercise we'll introduce the different ways that Ansible provides of supplying variables and the reasons for using them.
+
+## Exercise Learning Objectives
+In this exercise you'll learn how to use variables to:
+  - manage differences across systems
+  - keep code and configuration separate.
+  - organise your configuration.
+
+## Useful Terminology
+- DRY (Don't Repeat Yourself) - keeping code DRY is a very simple but important idea, that being that the duplication is best avoided so keep your code DRY and don't repeat yourself.
+
+**Note:** Before starting the exercises, please do the following:
+  - `cd YOUR_CLONE_OF_THIS REPO`
+  - `source ./bin/env`
+  - `cd <%=env('exercise_path') %>`
+  - `<%= command("cic up") %>`
+
+
+Having run this command, test infrastructure consisting of all the environments you will need has been stood up for you.
+
+Runing <%= after_rendering_run(cic_down_command) %> will stop these environments and drop any changes that you have made to them.
```

Comment:
As I suggested rewording the cic up text, then we could single-case 'these environments' to 'the environment' instead.

Reply:
Agreed